es-bs-base/README.md

```diff
@@ -4,7 +4,7 @@ An EnergySage branded Bootstrap
 
 ## Getting Started
 
-`npm install @energysage/es-bs-base`
+`npm install --save @energysage/es-bs-base`
```

Comment:
Doesn't `npm install` now save into dependencies by default, even without the `--save` param?
https://docs.npmjs.com/cli/v6/commands/npm-install

Reply:
I didn't know that was the default 👍, reverted via 0629cfd4203edacf63f88670f2bbcfb8f600d799